### DISCUSSION TITLE
Collection support brainstorm

### DISCUSSION THREAD

**Discussion by inmanturbo** [rappasoft/laravel-livewire-tables#432] [Feature Requests]
Hey @rappasoft 

I see that you have on the roadmap to add support for collections, I am opening this a place to discuss plans for the feature.

I was wondering, have you started any work on it yet, and what you might want a PR which adds this feature to look like?

Any thoughts or concerns, challenges or pitfalls worth mentioning? Or is is just something you are planning but haven't had a  chance to look into yet?

This is a feature I could really use, and I'm considering working on it, but I haven't done the "deep dive" with it in mind just yet :smile: . I wanted to see if you had any insights first, before I go off half-cocked.

**Comment by rappasoft** [comment]
I have not had the time to dive into it, but ideally, it would support all the features. We would have to see what features would be an issue using a collection over Eloquent so we can figure out how to best support them.

**Reply by inmanturbo** [reply]
Ok Cool. I'm working my way in that direction. I just have a lot on my plate here (as I'm sure we all do -- there is never enough time!). Besides, I'm trying to free up some time to take the boat out. :motor_boat:

**Comment by inmanturbo** [comment]
For anyone stumbling across here, I've actually started using https://github.com/calebporzio/sushi (eloquent array driver)
And just calling toArray on the collection inside the getRows method. The readme has details on how you can cache the rows, and bust cache as needed.

**Reply by michel2103** [reply]
Hello. Are you still working on it?
